src/Image/FaceDetection/FacePositionToGeometry.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Flyimg\Image\FaceDetection;
+
+use Flyimg\Image\ShortLivedStreamedFile;
+use Imagine\Image\Box;
+use Imagine\Image\ImageInterface;
+use Imagine\Image\Point;
+use Symfony\Component\Process\Process;
+
```

Comment:
It will be nice to have comment explaining what is the purpose of this class

Reply:
It is the object representing a file in `imagine/imagine`, an image manipulation library abstracting imagick, gmagick and gd2 behind an unique interface.

The former `InputImage` and `OutputImage` would disappear and the logic that makes sense to keep may be moved to some other place.